record/record.app.js

```diff
@@ -57,7 +58,59 @@
         context.pageId = MathUtils.uuid();
 
         ERMrest.appLinkFn(UriUtils.appTagToURL);
-
+        /* @desc getPageSize(obj) returms page size of the obj.           */
+        function getPageSize(obj){
+            return ((!angular.isUndefined(obj) && obj.display.defaultPageSize)?obj.display.defaultPageSize:constants.defaultPageSize);
+        }
+        /*@desc  getRelatedTableData(refObj, accordionOpen, callback) returns model object with all required component values
+        *       that is needed by <record-table>, <record-action-bar> and <record-display> diretives.
+        *       refObj: Reference object with component details
+        *       accordionOpen: bool value
```

Comment:
add `@param` and `{<type>}` here with the proper type of the expected attribute.